src/router/index.js

```diff
@@ -21,6 +21,7 @@ export default (async () => {
       return { x: 0, y: 0 };
     },
     routes: (await Promise.all([
+      import(/* webpackChunkName: "ui-mobile" */ './routes/common'),
```

Comment:
```suggestion
      import(/* webpackChunkName: "ui-common" */ './routes/common'),
```
otherwise won't it make desktop to depend on mobile routes?